invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentPendingLoans/state/actions.js

```diff
@@ -0,0 +1,25 @@
+import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
+import { loan as loanApi, serializeLoan } from '../../../../../../common/api';
+
+export const fetchPendingLoans = documentPid => {
+  return async dispatch => {
+    dispatch({
+      type: IS_LOADING,
+    });
+
+    await loanApi
+      .fetchLoans(documentPid, null, null, null, null, 'PENDING')
```

Comment:
For another refactoring, not related to your changes, what about have this nicer and use  a Builder pattern?
Like this you would have something like:
```
const loansFetcher = 
    LoansFetcher(documentPid)
        .withState('PENDING')
        .build();

await loansFetcher.fetch().then()....
```
In general it is a nice way to handle optional params.
It is also valid for Items and Documents (ping @zzacharo @topless if they have also thoughts about this)

Reply:
😍 I will add a ticket to refactor this, because I also don't like it. thanks!